features/openhab-addons/src/main/feature/feature.xml

```diff
@@ -404,6 +404,13 @@
     <configfile finalname="${openhab.conf}/services/mqtt-eventbus.cfg" override="false">mvn:${project.groupId}/openhab-addons-external/${project.version}/cfg/mqtt-eventbus</configfile>
   </feature>
 
+  <feature name="openhab-binding-mqttitude" description="Mqttitude (Owntracks) Binding" version="${project.version}">
```

Comment:
`OwnTracks (formerly MQTTitude) Binding` (since the old "UI-level" name is no longer used)